### PR TITLE
Setting up tmc-api-team environment

### PR DESCRIPTION
**Pull request by Rochet2**
Contains:

- bug fixes for running tests in docker.
- disable test that fails
- add additional docker yml files used by tmc api team
- add swagger-blocks to gems
- add swagger documentation page to routes and controllers

### CODE REVIEW

**Comment by jamo** [comment]
@testmycode/api-team testing

File: app/controllers/api/v8/apidocs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  include Swagger::Blocks
+
+  swagger_root do
+    key :swagger, '2.0'
+    info do
+      key :version, '1.0.0'
+      key :title, 'TMC API documentation'
+      key :description, 'TMC API documentation'
+      contact do
+        key :name, 'TMC API Team'
```
Comment (jamo) [inline app/controllers/api/v8/apidocs_controller.rb]: Should probably be TestMycode / Rage Research... cc @nygrenh

File: app/controllers/api/v8/apidocs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      end
+    end
+    tag do
+      key :name, 'api'
+      key :description, 'API operations'
+    end
+    parameter :path_organization_id do
+      key :name, :organization_id
+      key :in, :path
+      key :description, 'Organization\'s id'
```
Comment (jamo) [inline app/controllers/api/v8/apidocs_controller.rb]: maybe use `"` to avoid escaping `'`

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -88,7 +88,11 @@ def host_ip
 end
 
 # This makes it visible to others
-Capybara.server_host = host_ip
+if ENV['REMSAND']
```
Comment (jamo) [inline spec/spec_helper.rb]: ?

Comment (makevertti) [inline spec/spec_helper.rb]: This is required for running tests using remote sandbox. 'REMSAND' variable is set in docker compose files which use remote sandbox.

File: app/controllers/api/v8/apidocs_controller.rb (hunk 1 above)
Comment (nygrenh) [inline app/controllers/api/v8/apidocs_controller.rb]: TestMyCode is probably better

File: docker-compose-remote-sandbox-dev.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    environment:
+      - DB_HOST=db_dev
+      - HOST=$HOST
+      - SANDBOX_URL=$SANDBOX_URL
+    depends_on:
+      - db_dev
+    # allow running binding.pry for debugging
+    # see http://stackoverflow.com/questions/31669226/rails-byebug-did-not-stop-application
+    stdin_open: true
+    tty: true
```
Comment (nygrenh) [inline docker-compose-remote-sandbox-dev.yml]: Do you still need these? Any reason for not running the `web_dev` container interactively?

File: Gemfile [hunk 5]
```diff
@@ -33,6 +33,8 @@ gem 'logstasher', '~> 0.9.0'
 
 gem 'pghero'
 
+gem 'swagger-blocks'
```
Comment (nygrenh) [inline Gemfile]: Should we lock the dependency version?

**Pull request merged**